system-tests/lib/mock/utils.go

```diff
@@ -0,0 +1,79 @@
+package mock_capability
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+	pb2 "github.com/smartcontractkit/chainlink-common/pkg/values/pb"
+	jobv1 "github.com/smartcontractkit/chainlink-protos/job-distributor/v1/job"
+)
+
+func MapToBytes(m *values.Map) ([]byte, error) {
+	if m == nil {
+		return nil, nil
+	}
+
+	pm := make(map[string]*pb2.Value)
+	for k, v := range m.Underlying {
+		pm[k] = values.Proto(v)
+	}
+	bytes, err := proto.Marshal(pb2.NewMapValue(pm))
+	if err != nil {
+		return nil, err
```

Comment:
general comment: I do like using `errors.Wrap` and `errors.Wrapf` from the deprecated pkg/errors package, because they give you stacktrace. It is very commonly used in this module.